src/Orders.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+import Button from './Button'
+class Orders extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      orders: 0
+    }
+  }
+  addOrder = () => {
+    this.setState(previousState => {
+      return { orders: previousState.orders + 1 }
+    })
+  }
+  render () {
+    return (
+      <ul>
+        <li>
+          {this.props.orderType}: {this.state.orders}{' '}
+          <Button caption='Add' addOrder={this.addOrder} />
```

Comment:
Great job recognising that we had already implemented a `Restaurant` button component. Do you see how we're passing the `this.addOrder` function as a prop, which is then called by `Button` when it is clicked?